### PR TITLE
[type: refactor #3088] Format data-permission-sqlmap.xml

### PR DESCRIPTION
**Pull request by Mayank-KS**
Format SQL to improve code readability in shenyu-admin.
Format data-permission-sqlmap.xml

### CODE REVIEW

File: shenyu-admin/src/main/resources/mappers/data-permission-sqlmap.xml [hunk 1]
```diff
@@ -28,24 +28,34 @@
...
+        date_updated, 
+        user_id, 
+        data_id, 
+        data_type
     </sql>
 
     <delete id="deleteByUniqueKey">
-        delete from data_permission where data_id = #{dataId, jdbcType=VARCHAR}
-        and user_id = #{userId,jdbcType=VARCHAR} and data_type = #{dataType,jdbcType=INTEGER}
+        delete from data_permission
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/data-permission-sqlmap.xml:40]: - The keywords  should be Uppercase;
-  Keep keywords right aligned；
-  Keep fields left aligned；

**Comment by Mayank-KS** [comment]
done with the changes

**Review by midnight2104 — APPROVED** [review]

**Comment by midnight2104** [comment]
Good job. If you want, you can add me on WeChat `LL332595980` and invite you to join the shenyu contributor group.

**Comment by Mayank-KS** [comment]
WeChat is banned in my country, so is there any other platform where I can join and interact like slack, discord or telegram?

**Comment by Mayank-KS** [comment]
@midnight2104 please give some update

**Pull request merged**